### PR TITLE
Correct announcement of move when variations selected

### PR DESCRIPTION
**Pull request by ukscid**
Move was announced when the variation selection dialog popup. Now announce move after the selection with announcing the selected move.
Move was announced twice when the variation selection dialog was not used.

### CODE REVIEW

File: tcl/main.tcl [hunk 1]
```diff
@@ -545,6 +545,7 @@ proc showVars {} {
         } else  {
             sc_var moveInto [expr $cur - 1]
         }
+        ::utils::sound::AnnounceForward [sc_game info previous]
```
Comment (benini) [inline tcl/main.tcl]: It would be better to invoke ::move::Follow instead of manually entering into the var, announcing the move and updating the board.

**Comment by benini** [comment]
I have added a commit on how I think it should work.
Can you please try it and eventually fix it?

File: tcl/main.tcl (hunk 1 above)
Comment (ukscid) [reply tcl/main.tcl]: Do you mean to rewrite `bind .variations <Right>` and use therefor  `::move::Follow` ?
move::Follow needs a uci move as input. Where should this come from?

**Comment by ukscid** [comment]
I checked and tested your code and made small adjustment. This works.

File: tcl/main.tcl (hunk 1 above)
Comment (benini) [reply tcl/main.tcl]: There are multiple ways to enter into a variation (entering the move with the mouse or the keyboard, the varPopup, an external board, etc...) andi t is desireable to have the same behavior for all of them and use the same code.

I was thinking about replacing
$w.lbVar insert {} end -id $j ...
with
$w.lbVar insert {} end -id $move ...
and the just
::move::Follow [.variations.lbVar selection]

However it probably does not work if there are variations that start with the same move.

**Pull request merged**